vars/runInspector.groovy

```diff
@@ -28,109 +29,228 @@ import java.util.*
 
 
 def call(body) {
+    def stackName = 'InspectorAmiTest' + UUID.randomUUID().toString()
+    def bucketName = 'inspectortestbucket' + UUID.randomUUID().toString()
+    def fileName = 'Inspector.yaml'
+    def findings = ''
+    try{
+        findings = main(body, stackName, bucketName, fileName)
+    } catch(Exception e) {
+        println(e)
+        cleanUp(stackName, body.region, bucketName, fileName)
+        return 1
```

Comment:
rather than returning the value of 1 here, just throw the same exception after the cleanup

```groovy
} catch(Exception e) {
        print('unexpected event occurred when running inpsector, cleaning up the stack')
        cleanUp(stackName, body.region, bucketName, fileName)
        throw e
}
```

Reply:
fixed